File: code_corpus/manual/sort_3.m
function dataOut = bubbleSort(data)
lenD = size(data,2);
for i = 1:lenD
  for j = (lenD):-1:(i+1)
    if(data(j)<data(j-1))
      tmp = data(j);
      data(j)=data(j-1);
      data(j-1)=tmp;
    end
  end
end
dataOut = data;

File: code_corpus/manual/sort_3.cpp
void SortAlgo::bubbleSort(int data[], int lenD)
{
  int tmp = 0;
  for(int i = 0;i<lenD;i++){
    for(int j = (lenD-1);j>=(i+1);j--){
      if(data[j]<data[j-1]){
        tmp = data[j];
        data[j]=data[j-1];
        data[j-1]=tmp;
      }
    }
  }
}
